spec/index.md

```diff
@@ -5089,7 +5187,7 @@ S with
 
 ```
 
-If the responded call context does not exist anymore, because the canister has been uninstalled since, the refunded cycles are still added to the canister balance, but no function invocation is enqueued:
+If the responded call context does not exist anymore, because the canister has been uninstalled since, the refunded cycles are still added to the canister balance, but no function invocation is enqueued. No refunds are issued for expired calls.
```

Comment:
```suggestion
If the responded call context does not exist anymore, because the canister has been uninstalled since, the refunded cycles are still added to the canister balance, but no function invocation is enqueued.
```
This follows from setting the refunded cycles to zero when expiring the corresponding message. I wouldn't point it out here since it's confusing (there's no special treatment of expired cycles here in terms of refunded cycles).

Reply:
OK, I'll move the remark to the expiry transition.